EmployeeRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.ifour.EmployeeManagement.Employee;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface EmployeeRepository extends JpaRepository<Employee,Integer> {

```

Comment:
Sorry if I am wrong, but didn't project requirements say that we will use simple JDBC and not spring data package?